packages/lit-dev-content/site/docs/ssr/server-api.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Lit SSR server API
+eleventyNavigation:
+  key: Server API
+  parent: Server-side rendering
+  order: 2
+---
+
+Lit SSR provides a couple different ways of rendering custom elements server-side: rendering in the [global scope](#global-scope) or via [VM modules](#vm-module).
+
+In order to render custom elements in Node, Lit SSR requires some minimal [DOM emulation]('/docs/ssr/dom-emulation') necessary for `lit-html` and `reactive-element` to work, as well as a working `CustomElementRegistry`.
+
+When rendering in the global scope, this DOM shim is installed in the global space. This may cause unintended behaviors for other libraries which, for instance, might try to detect the running environment by checking for the presence of `window` in the global space. All render requests also share the same global custom element registry, and any data that might be stored globally.
+
+Rendering with VM modules allow each render request to have its own context with a separate global from the main Node process such that the DOM emulation is only installed within that scope and any custom elements being registered or global data is contained within that context used only for that particular render request. It does require utilizing an experimental node feature with a particular way of using it. See below for details.
```

Comment:
I'd say the usage is slightly less idiomatic since instead of calling a function directly, you have to set up a separate module file, with a function within it, and specify the function name to call within that. Perhaps it's best to leave that portion to the table though.

Reply:
Got it. Yeah, I'd say either leave it to the table or be a little more concrete, like, "Rendering with VM modules requires a little extra boilerplate code."